pandas/io/formats/excel.py

```diff
@@ -56,15 +61,15 @@ class CSSToExcelConverter:
     #     instancemethods so that users can easily experiment with extensions
     #     without monkey-patching.
 
-    def __init__(self, inherited=None):
+    def __init__(self, inherited: Optional[str] = None):
         if inherited is not None:
             inherited = self.compute_css(inherited)
 
         self.inherited = inherited
 
     compute_css = CSSResolver()
 
-    def __call__(self, declarations_str: str):
+    def __call__(self, declarations_str: str) -> Dict[str, Dict]:
```

Comment:
This annotation as a return value would mean that this *always* needs to be a dict of dicts - is that true?

Reply:
Correct.

Because it's returning at the end

```python
return self.build_xlstyle(properties)
```

and ```build_xlstyle``` is returning  a ```Dict``` containing:

```python
out = {
    "alignment": self.build_alignment(props),
    "border": self.build_border(props),
    "fill": self.build_fill(props),
    "font": self.build_font(props),
    "number_format": self.build_number_format(props),
}

.........

return out
```

and each one of 

```[build_alignment, build_border, build_fill, build_font, build_number_format]```

also returns a ```Dict```

---

This is how I see it, you're way more experienced than I am,  if you think otherwise I'd like to hear it.

Reply:
Ah OK I see makes sense. So can we go further and do `Dict[str, Dict[str, str]]` to describe this exactly again? Subtypes are much more useful compared to just typing as `Dict`